### ISSUE TITLE
gantt not actually working

### ISSUE THREAD

**Issue by AlbinoGeek** [benweet/stackedit#1701]
Contrary to #1137 being closed, the examples do not render.

https://mermaid-js.github.io/mermaid/

```mermaid
gantt
dateFormat	YYYY-MM-DD
title		Adding GANTT diagram to mermaid
excludes	weekdays	2014-01-10

section A section
Completed task 		:done, 		des1, 2014-01-06,2014-01-08
Active task 		:active, 	des2, 2014-01-09, 3d
Future task 		: 		des3, after des2, 5d
Future task2 		: 		des4, after des3, 5d
```

[ Yes, I've tried without any whitespace, with only spaces, with tabs, etc. ]

**Comment by julirehe** [comment]
@AlbinoGeek,

I've faced the same issue, however the example from diagrams.net works fine for me:
https://www.diagrams.net/blog/mermaid-diagrams

`gantt
 title Example Gantt diagram
    dateFormat  YYYY-MM-DD
    section Team 1
    Research & requirements :done, a1, 2020-03-08, 2020-04-10
    Review & documentation : after a1, 20d
    section Team 2
    Implementation      :crit, active, 2020-03-25  , 20d
    Testing      :crit, 20d`

Maybe it's an enconding or whitespace character problem with the other example. Haven't sorted it out and won't do that for now, since it's working.

**Comment by AlbinoGeek** [comment]
> @AlbinoGeek,
> 
> `gantt title Example Gantt diagram dateFormat YYYY-MM-DD section Team 1 Research & requirements :done, a1, 2020-03-08, 2020-04-10 Review & documentation : after a1, 20d section Team 2 Implementation :crit, active, 2020-03-25 , 20d Testing :crit, 20d`

That syntax (nor any combination of whitespace attempted) works on StackEdit. Only the title is recognized. Also, diagrams I generate in the mermaid editor do not work "as is" (copy-pasted) into StackEdit.

This leads me to believe that StackEdit is using an older version of mermaid and hasn't updated to syntax fixes/the new item syntax.

The only parts of your syntax I could make work on StackEdit were (although, you didn't quote your special characters, ampersands, etc, so that could easily be at fault.):

![image](https://user-images.githubusercontent.com/1910461/110848852-656dcd00-8263-11eb-870f-44e843a557db.png)

**Comment by benweet** [comment]
Fixed with v5.14.10.
